src/main/java/org/ml_methods_group/plugin/AutomaticRefactoringAction.java

```diff
@@ -94,6 +95,14 @@ public static AutomaticRefactoringAction getInstance(@NotNull Project project) {
 
     @Override
     protected void analyze(@NotNull final Project project, @NotNull final AnalysisScope analysisScope) {
+        if (analysisScope.getFileCount() == 0) {
```

Comment:
This works fine with new and empty projects. For a non-empty project, if I open `Project Structure` -> `Modules`, and deselect all `Sources` (and `Tests`) marks, `analysisScope.getFileCount()` is not 0. We'd better use `analysisScope.getFiles().isEmpty()`, is seems to work more predictable